### PR TITLE
v1 account move resources 

### PR DESCRIPTION
**Pull request by evilpeach**
- [x] Implement the account move resource route
- [x] Implement the `get_move_resources` request function
- [x] Write a unit test

### CODE REVIEW

File: server/adapters/move/resources.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import requests
+from flask import abort
+from utils.gcs import get_network_data
+
+
+def get_move_resources(chain: str, network: str, address: str):
+    if chain != "initia":
+        return abort(404, f"invalid chain ({chain}).")
```
Comment (songwongtp) [inline server/adapters/move/resources.py:8]: just `abort` no return needed

File: server/tests/test_accounts.py [hunk 2]
```diff
@@ -184,3 +184,24 @@ def test_get_codes():
...
+
+    response_json = response.json
+
+    for item in response_json["items"]:
+        assert type(item["address"]) == str
+        assert type(item["move_resource"]) == str
+        assert type(item["raw_bytes"]) == str
+        assert type(item["struct_tag"]) == str
+
+    assert type(response_json["total"]) == int
```
Comment (songwongtp) [inline server/tests/test_accounts.py:207]: add invalid chain case

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: server/routes/v1/accounts.py [hunk 3]
```diff
@@ -179,3 +180,11 @@ def get_codes(chain, network, account_address):
     del data["codes_aggregate"]
 
     return data
+
+
+@accounts_bp.route(
```
Comment (songwongtp) [inline server/routes/v1/accounts.py:185]: should this be under `move` route?

i.e., `/<chain>/<network>/move/resources/<account_address>`

Comment (evilpeach) [reply server/routes/v1/accounts.py:185]: As we discussed, let use account path for now.

**Review by songwongtp — APPROVED** [review]

**Pull request merged**